### PR TITLE
User can see their registered events- Schedule (Integrated Firebase) - Fixes #21

### LINKED ISSUE #21: User filters events to see where they volunteered.
## Description
As an attendee,
I want to view the events I have signed up as a Volunteer,
so that I can plan accordingly and review responsibilities before the event.

## Mocks
TBD @pandilladivya Can you include a mock screen below?

## Acceptance Criteria
- User is able to view volunteer status on small popup of events on the map. 
- On the detailed events screen, they see volunteer status.
- User can filter the map to show only events they registered for as volunteers.

## Definition of Done
- Passes all regression tests.
- Passes testing per acceptance criteria items.
- Able to show feature in demo.

## Estimation
- 8 Hours

### PR DESCRIPTION
**Pull request by pandilladivya**
The following is the demo of the work done.
ezgif com-video-to-gif 13

### CODE REVIEW

**Comment by HimanshiJain** [comment]
@pandilladivya dependent on #71 right? It's better to mention such cases.

**Comment by HimanshiJain** [comment]
Each Scheduled event should also reflect the role the user has registered for.

**Comment by pandilladivya** [comment]
screen shot 2017-08-20 at 4 00 37 pm


@HimanshiJain @sandarumk 
Added Attendee type to the scheduled event.

**Comment by pandilladivya** [comment]
Will resolve conflicts as soon as #71 is merged.

**Comment by pandilladivya** [comment]
Resolved conflicts by doing a rebase.
@sandarumk @HimanshiJain 
Kindly Review.

File: app/src/main/res/layout/schedule_list_fragment.xml [hunk 1]
```diff
@@ -11,4 +11,13 @@
         android:divider="@null"
         android:dividerHeight="@dimen/padding"
         />
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
+        android:text="@string/no_registrations"
+        android:id="@+id/noeventslabel"
```
Comment (pandilladivya) [inline app/src/main/res/layout/schedule_list_fragment.xml]: Okay.

**Comment by pandilladivya** [comment]
@HimanshiJain @sandarumk 
Done with the changes.
Kindly review! :)

**Pull request merged**